tool/tsh/kube.go

```diff
@@ -638,12 +638,14 @@ type kubeLSCommand struct {
 	predicateExpr  string
 	searchKeywords string
 	format         string
+	siteName       string
 }
 
 func newKubeLSCommand(parent *kingpin.CmdClause) *kubeLSCommand {
 	c := &kubeLSCommand{
 		CmdClause: parent.Command("ls", "Get a list of kubernetes clusters"),
 	}
+	c.Flag("cluster", clusterHelp).Short('c').StringVar(&c.siteName)
```

Comment:
It's a little strange to me that we have the `-c` abbreviation for the kube commands but not for any of the other `tsh` commands. Might be something to fix in the future.

Reply:
I created an issue to track that decision